reference/shipping.v2.yml

```diff
@@ -1189,6 +1195,25 @@ paths:
           "shipping_cost": 8
         }
         ```
+        
+        ### Channels
+
+        Example request body: 
+
+        ```json
+        {
+          "name": "Method associated to channels 1,3",
+          "type": "peritem",
+          "settings": {
+            "rate": 5
+          },
+          channels: [1, 3]
+        }
+        ```
+
+        | Property | Type | Description |
+        |:---------|:-----|:------------|
+        | channels | array | channels associated with the method as an array of integers |
```

Comment:
```suggestion
        | channels | array | Channels associated with the method as an array of integers |
```

Reply:
Same as this one, updated the others as well. 🍻